### DISCUSSION TITLE
Support Authentication on the Emulator

### DISCUSSION THREAD

**Discussion by tomhicks** [invertase/react-native-firebase#4463] [Ideas] [answered]
Yesterday, Firebase released support for Authentication on the Emulator.

https://firebase.google.com/docs/emulator-suite/connect_auth

It seems like a fairly straightforward bit of API to expose with a simple `useEmulator` call on the auth instance.

Support for this would be welcomed!

**Comment by andersonaddo** [comment, marked as answer]
https://github.com/invertase/react-native-firebase/discussions/4461

**Reply by mikehardy** [reply]
@tomhicks PRs to add the support would be welcomed :-), otherwise as mentioned in the dupe link, it might be a bit...

**Reply by mikehardy** [reply]
v10, just released, has auth emulator support! Enjoy
